setup.py

```diff
@@ -31,6 +31,8 @@ def readfile(filename):
         'lstmcpipe_plot_models_importance = lstmcpipe.plots.plot_models_importance:main',
         'lstmcpipe_plot_irfs = lstmcpipe.plots.plot_irfs:main',
         'lstmcpipe_dl2_to_sensitivity = lstmcpipe.stages.script_dl2_to_sensitivity:main'
+        'lstmcpipe_utils_move_dir = lstmcpipe.stages.script_merge_utils_move_dir:main',
```

Comment:
Should these have their own entry points?
Looks like they could just be functions, that are called from the stages or main script

Reply:
I think they should.
These stages are quite useless (I'm sure this can be done somehow), but I didn't find another way of performing these actions (move DL1 files, copy config and move DL1 dir) **after** the merging has succeeded. If they are performed before, the pipeline will not work because no "final" DL1 file has been merged. Also, the end of these jobs will allow the training to start (If no 'final' DL1 file in the correct directory, the training won't work).

Reply:
The only solution to solve this (I think) is to completely remove the `running_analysis` stage (comes from some years ago), and create and merge everything in the same DL1 dir.

Maybe another point to improve ? ^^

Reply:
Ok, then lets keep it like that for now

Reply:
There is a comma missing in l33 btw.
The script is not getting installed